scripts/l2systemDeploy.ts

```diff
@@ -0,0 +1,476 @@
+import hre, { ethers } from 'hardhat';
+import { expect } from 'chai';
+import config from './config';
+import {
+  L2Core,
+  OraclePassThrough,
+  PCVGuardAdmin,
+  PCVGuardian,
+  PriceBoundPSM,
+  L2ScalingPriceOracle
+} from '@custom-types/contracts';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+
+const {
```

Comment:
Having these vars be set by order / destructuring seems like a recipe for overlooking something. Would suggest manually specifying each var.

Reply:
Manually specifying each variable here via env or hardcoding directly into this file seems error prone. Once PR 80 merges to develop, I will merge dev into this branch, and use mainnetAddresses instead of this setup.

Reply:
Wasn't recommending via env or hardcoding; just meant not using destructuring

Reply:
so each variable is imported from the config file on its own line instead of the current import structure?